### PR TITLE
adds the about fragment using external library and an attribute edita…

### PR DESCRIPTION
**Pull request by yanisepfl**
…ble to posts

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/kandle/Post.java [hunk 1]
```diff
@@ -152,6 +153,14 @@ public void setLatitude(double latitude) {
         this.latitude = latitude;
     }
 
+    public Boolean getEditable(){
```
Comment (AntoineSidem) [inline app/src/main/java/ch/epfl/sdp/kandle/Post.java:156]: Getters for booleans start with `is` → `isEditable`

File: app/src/main/java/ch/epfl/sdp/kandle/PostAdapter.java [hunk 2]
```diff
@@ -124,9 +125,12 @@ public void onBindViewHolder(@NonNull final ViewHolder holder, int position) {
                 //milliseconds
                 long different = new Date().getTime() - post.getDate().getTime();
                 long minutes = different / 60000;
-                if(user.getId().equals(userId)){
+                if (user.getId().equals(userId)) {
                     deletePostView.setVisibility(View.VISIBLE);
-                    if (minutes < 6) {
+                    if (minutes >= 6) {
```
Comment (AntoineSidem) [inline app/src/main/java/ch/epfl/sdp/kandle/PostAdapter.java:130]: Magic value 6, make a constant

File: app/src/main/java/ch/epfl/sdp/kandle/fragment/AboutFragment.java [hunk 3]
```diff
@@ -1,24 +1,42 @@
...
 
 public class AboutFragment extends Fragment {
-/*
+
     @Override
     public View onCreateView(LayoutInflater inflater,ViewGroup container, Bundle savedInstanceState) {
-        // Inflate the layout for this fragment
-        View view = inflater.inflate(R.layout.fragment_search, container, false);
+        Element versionElement = new Element().setTitle("Beta Version 1.0.0");
+        Element adsElement = new Element().setTitle("Advertises");
```
Comment (AntoineSidem) [inline app/src/main/java/ch/epfl/sdp/kandle/fragment/AboutFragment.java:23]: Did you mean "Advertisements"?

File: app/src/main/java/ch/epfl/sdp/kandle/fragment/AboutFragment.java [hunk 4]
```diff
@@ -1,24 +1,42 @@
...
+import mehdi.sakout.aboutpage.Element;
 
 public class AboutFragment extends Fragment {
-/*
+
     @Override
     public View onCreateView(LayoutInflater inflater,ViewGroup container, Bundle savedInstanceState) {
-        // Inflate the layout for this fragment
-        View view = inflater.inflate(R.layout.fragment_search, container, false);
+        Element versionElement = new Element().setTitle("Beta Version 1.0.0");
```
Comment (AntoineSidem) [inline app/src/main/java/ch/epfl/sdp/kandle/fragment/AboutFragment.java:22]: Strings are better in the resources files

File: app/src/main/java/ch/epfl/sdp/kandle/Post.java (hunk 1 above)
Comment (yanisepfl) [reply app/src/main/java/ch/epfl/sdp/kandle/Post.java:156]: Indeed, thanks

File: app/src/main/java/ch/epfl/sdp/kandle/PostAdapter.java (hunk 2 above)
Comment (yanisepfl) [reply app/src/main/java/ch/epfl/sdp/kandle/PostAdapter.java:130]: Good, will do that, as well as for 60000

**Pull request merged**